numpy/core/include/numpy/ufuncobject.h

```diff
@@ -130,8 +130,8 @@ typedef struct _tagPyUFuncObject {
         /* The number of elements in 'functions' and 'data' */
         int ntypes;
 
-        /* Used to be unused field 'check_return' */
-        int reserved1;
+        /* Used to be unused field 'check_return', repurposed in 1.16 */
+        const int version;
```

Comment:
This is outdated now, right?

Reply:
In `PyUFunc_FromFuncAndDataAndSignature` it is set to `NPY_API_VERSION`, which is then checked in `PyUFunc_GeneralizedFunction` (via a call to `_initialize_variable_parts`) to make sure it is `>= 0x00000d`

Reply:
This seems pointless to me

Reply:
Removing. If it becomes an issue we can revisit this.